### DISCUSSION TITLE
GCP: How to create CloudSQL Replica

### DISCUSSION THREAD

**Discussion by 0xSalman** [pulumi/pulumi#9023] [Pulumi in Practice] [answered]
hello everyone, 

I am looking to deploy postgres on GCP with cross region replicas. I wasn’t able to find anything in the docs. Are cross-region replicas possible with pulumi?

Thank you!

**Comment by gfaithSC** [comment, marked as answer]
This might be what you are looking for:
https://www.pulumi.com/registry/packages/gcp/api-docs/sql/databaseinstance/#databaseinstancereplicaconfiguration

**Reply by 0xSalman** [reply]
Thank you for sharing that. I knew about that one but didn't use it because the docs says `Valid only for MySQL instances`. I guess I could try it for postgres and see if it works

**Reply by nimbinatus** [reply]
Thanks, @gfaithSC! Did this answer help you, @salman-ahmad?
